src/Components/SavedResponses.js

```diff
@@ -0,0 +1,95 @@
+// Modules
+import firebase from '../firebase';
+import { useEffect, useState } from 'react';
+import { getDatabase, ref, onValue } from 'firebase/database';
+import { FaRegHeart, FaRegSmile, FaRegFrown, FaRegMeh, FaArrowUp } from 'react-icons/fa';
+
+const PastResponses = () => {
+  // Defining State
+  const [userData, setUserData] = useState([]);
+  const [count, setCounter] = useState(6);
+  // Firebase Connection
+  useEffect(() => {
+    // Variable to hold database details:
+    const database = getDatabase(firebase);
+    // Variable that references our database:
+    const dbRef = ref(database);
+    // Event listener that will fire from the database & call the data 'response'
+    onValue(dbRef, (response) => {
+      // Variable to store the new state we're creating in our app
+      const newState = [];
+      // Store our response,val() in a variable;
+      const dataResponse = response.val();
+      // Data is an object so we iterate using a for-in loop to access each tarot card object.
+      for (let key in dataResponse) {
+        // Push each tarot card object to an array we already created in newState
+        newState.push({ key: key, tarotCard: dataResponse[key] });
+      }
+      // Call setUserData to update our components state using the local array newState:
+      setUserData(newState);
+    })
+  }, [])
+
+  // Show More Results Button
+  const handleShowMore = (event) => {
+    event.preventDefault();
+    const addResults = count + 6 >= 18 ? 18 : count + 6;
+    setCounter(addResults);
+  }
+
+  // Show Less Results Button
+  const handleShowLess = (event) => {
+    event.preventDefault();
+    const subtractResults = count - 6;
+    setCounter(subtractResults);
+  }
+
+  return (
+    <section id='saved-responses' className='outer-container saved-responses'>
+      <div className='wrapper'>
+        <h2 className='saved-response-heading'>Saved Responses</h2>
```

Comment:
a nice way to sauce on characters would be:
<h2 className="saved-response"> and then in css h2.saved-response

Reply:
Yes! I was struggling w/ how to name these.. great suggestion.